jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/upload/LambdaConfigPanelTest.kt

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.upload
+
+import com.intellij.testFramework.ProjectRule
+import com.intellij.testFramework.runInEdtAndWait
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import software.amazon.awssdk.services.iam.model.Role
+import software.amazon.awssdk.services.lambda.model.Runtime
+import software.aws.toolkits.core.utils.test.aString
+import software.aws.toolkits.jetbrains.core.MockResourceCacheRule
+import software.aws.toolkits.jetbrains.services.iam.IamResources
+import software.aws.toolkits.jetbrains.services.iam.IamRole
+import software.aws.toolkits.jetbrains.utils.waitToLoad
+
+class LambdaConfigPanelTest {
```

Comment:
very minor nit: we could test negative values as well

Reply:
Added